packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -353,58 +282,26 @@ export class Expression
                     </LabelSmall>
                 )}
                 {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions -- TODO(LEMS-2871): Address a11y error */}
-                <div
-                    className={className}
-                    onBlur={() =>
-                        this.state.invalid &&
-                        this.setState({
-                            showErrorTooltip: true,
-                            showErrorStyle: true,
-                        })
-                    }
-                    onFocus={() =>
-                        this.setState({
-                            showErrorTooltip: false,
-                        })
-                    }
-                >
-                    {/**
-                * This is a visually hidden container for the error tooltip.
-                https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role#example_3_visually_hidden_alert_container_for_screen_reader_notifications
-            */}
-                    <View style={a11y.srOnly} role="alert">
-                        {this.state.showErrorTooltip &&
-                            ERROR_TITLE + " " + ERROR_MESSAGE}
-                    </View>
-                    <Tooltip
-                        forceAnchorFocusivity={false}
-                        opened={this.state.showErrorTooltip}
-                        title={ERROR_TITLE}
-                        content={ERROR_MESSAGE}
-                    >
-                        <MathInput
-                            // eslint-disable-next-line react/no-string-refs
-                            ref="input"
-                            value={this.props.userInput}
-                            onChange={this.changeAndTrack}
-                            convertDotToTimes={this.props.times}
-                            buttonSets={this.props.buttonSets}
-                            onFocus={this._handleFocus}
-                            onBlur={this._handleBlur}
-                            hasError={this.state.showErrorStyle}
-                            ariaLabel={
-                                this.props.ariaLabel ||
-                                this.context.strings.mathInputBox
-                            }
-                            extraKeys={
-                                this.props.keypadConfiguration?.extraKeys
-                            }
-                            onAnalyticsEvent={
-                                this.props.analytics?.onAnalyticsEvent ??
-                                (async () => {})
-                            }
-                        />
-                    </Tooltip>
+                <div className={className}>
```

Comment:
Could this be simplified now?

```suggestion
                <div className="perseus-widget-expression">
```